5-WebApp-AuthZ/5-2-Groups/Startup.cs

```diff
@@ -38,24 +38,13 @@ public void ConfigureServices(IServiceCollection services)
             // Sign-in users with the Microsoft identity platform
             services.AddMicrosoftWebAppAuthentication(Configuration)
                     .AddMicrosoftWebAppCallsWebApi(Configuration, new string[] { "User.Read", "Directory.Read.All" })
-
-            // If you want to use group ids/names in the Authorize attribute then uncomment the following lines:
-            //services.Configure<OpenIdConnectOptions>(options =>
-            //        {
-            //            // Uncomment the following lines code instruct the asp.net core middleware to use the data in the "groups" claim in the [Authorize] attribute and for User.IsInrole()	
-            //            // See https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles for more info.	
-            //            // Use the groups claim for populating roles	
-            //            options.TokenValidationParameters.RoleClaimType = "groups";
-            //        })
-
                     .AddInMemoryTokenCaches();
 
             services.AddMSGraphService(Configuration);
 
             services.Configure<OpenIdConnectOptions>(OpenIdConnectDefaults.AuthenticationScheme, options => {
-                // Uncomment the following lines code instruct the asp.net core middleware to use the data in the "groups" claim in the [Authorize] attribute and for User.IsInrole()
+                // The following code instructs the ASP.NET Core middleware to use the data in the "groups" claim in the [Authorize] attribute and for User.IsInRole()
                 // See https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles for more info.
-                // Use the groups claim for populating roles
                 options.TokenValidationParameters.RoleClaimType = "groups";
```

Comment:
I think that given there is a groups claims we'd better offer in Microsoft identity we method to verify that things are in groups, rather than doing a hack of mapping the claim so that it can be used in the Authorize attribute?
What do you think, @pmaytak ? @jennyf19 ? @kalyankrishna1 @TiagoBrenck @Shama-K ?